services/prometheus/internal/common/config/http_config.go

```diff
@@ -14,7 +14,7 @@
 // You should have received a copy of the GNU Affero General Public License
 // along with this program. If not, see <https://www.gnu.org/licenses/>.
 
-// Original file: https://github.com/prometheus/common/blob/v0.2.0/config/http_config.go
+// Original file: https://github.com/prometheus/common/blob/v0.9.1/config/http_config.go
```

Comment:
https://github.com/percona/pmm-managed/blob/252ec20e62869842849bc208ac41ec3bf186b9cb/Gopkg.lock#L296-L306

@artemgavrilov Please update it in a separate PR.

Reply:
https://github.com/percona/pmm-managed/pull/348